docs/05-technical-reference/03-architecture/evnt-01-architecture.md

```diff
@@ -3,7 +3,9 @@ title: Eventing Architecture
 type: Architecture
 ---
 
-Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller which work together to process and deliver events in Kyma.
+Eventing uses NATS to implement the Event Publisher Proxy and the Eventing Controller which work together to process and deliver events in Kyma. See how Eventing works inside a Kyma cluster:
+
+![Eventing architecture](./assets/evnt-architecture.svg)
```

Comment:
Please make the connectors and outlines more visible (use HEX mentioned in the [draft](https://github.com/kyma-project/community/blob/4e619fac18aadb1005927f1a5b77b5c0a576b684/guidelines/content-guidelines/02-diagrams.md#connectors) guidelines for diagrams). Also, add a reference key below to explain the difference between the dotted and normal lines. For the future, consider adding the description for the steps.

```
1. Kyma user creates the Application CR.
2. Application Operator watches for Application CRs.
...
and so on
```